### DISCUSSION TITLE
Mriscoc Firmware on Ender 3 S1 Pro with S1 screen

### DISCUSSION THREAD

**Discussion by ImMaxe** [mriscoc/Ender3V2S1#1380] [Q&A]
Hi,
I recently installed the Mriscoc Professional Firmware on my Ender 3 S1 Pro (with CR-Touch and DACAI S1 screen). Everything seems to work, but I've noticed that the printer sounds different compared to the stock firmware. Is this normal?

Additionally, are there any specific steps or settings I should adjust to ensure everything runs smoothly with this firmware?

Thanks in advance for the help!

**Comment by mriscoc** [comment]
You must decrease speed/acceleration limits for a more quiet work.

**Reply by ImMaxe** [reply]
I took the configuration.h for the ender 3 S1 Pro and made ChatGpt find the parameters so I changed a few settings and I'm currently printing so it works great 😃
